docs/src/pages/backend-adapters/express.mdx

```diff
@@ -0,0 +1,166 @@
+import { Callout, Steps } from "nextra-theme-docs";
+
+# Getting started with Express
+
+## Package Setup
+
+<Steps>
+
+### Install the package
+
+import { Tab, Tabs } from "nextra-theme-docs";
+
+{/* prettier-ignore-start */}
+
+<Tabs items={["npm", "yarn", "pnpm"]}>
+  <Tab>
+    ```bash copy 
+    npm install uploadthing
+    ```
+  </Tab>
+  <Tab>
+    ```bash copy 
+    yarn add uploadthing
+    ```
+  </Tab>
+  <Tab>
+    ```bash copy 
+    pnpm install uploadthing
+    ```
+  </Tab>
+</Tabs>
+
+{/* prettier-ignore-end */}
+
+### Add env variables
+
+```bash copy
+UPLOADTHING_SECRET=... # A secret key for your app (starts with sk_live_)
+UPLOADTHING_APP_ID=... # Your app id
+```
+
+<Callout type="warning">
+  If you don't already have a uploadthing secret key, [sign up](https://uploadthing.com/sign-in) and create one from
+  the [dashboard!](https://uploadthing.com/dashboard)
+</Callout>
+</Steps>
+
+## Set Up A FileRouter
+
+<Steps>
+
+### Creating your first FileRoute
+
+All files uploaded to uploadthing are associated with a FileRoute. Think of a
+FileRoute similar to an endpoint, it has:
+
+- Permitted types ["image", "video", etc]
+- Max file size
+- (Optional) `middleware` to authenticate and tag requests
+- `onUploadComplete` callback for when uploads are completed
+
+```ts copy filename="src/uploadthing.ts"
```

Comment:
i'm not sure what the best user experience is, but it seems like a hassle to duplicate this 40 line codeblock and make minor changes to it. maybe we should have 1 doc explaining how to make a file router for all adapters, and then in each adapters doc just explain what's special with this (different req/res in middlewares)?

adding new stuff requires a lot of doc changes if this is duplicated too much i feel like